src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -298,47 +339,51 @@ FROM <repository>/<image>:<tag>
+If you are running a private registry internal to your company, you can https://docs.docker.com/registry/recipes/mirror/[optionally mirror DockerHub]. Your private registry will download and cache any images that your users reference from the public DockerHub. You must https://docs.docker.com/registry/recipes/mirror[configure your Docker daemon to make use of mirroring].
 
 [id="using-docker-in-workspaces"]
-== Using Docker In Workspaces
+== Using Docker in workspaces
 
-If you’d like your users to work with projects which have their own Docker images and Docker build capabilities inside of their workspace, then you need to configure the workspace to work with Docker. You have three options:
+To allow your users to work with projects that have their own container images and Docker build capabilities inside of their workspaces, you must configure the workspaces to work with Docker. Following are the three options to configure the workspaces:
 
-1.  Activate Docker privileged mode, where your user workspaces have access to the host.
+* Activate the Docker privileged mode where your user workspaces have access to the host.
 
 ----
 # Update your codenvy.env to allow all Che workspaces machines/containers privileged rights:
 CHE_DOCKER_PRIVILEGED=true;
 ----
 
-1.  Configure Che workspaces to volume mount the host docker daemon socket file.
+* Configure Che workspaces to mount the host Docker daemon socket file.
 
 ----
 # Update your codenvy.env to allow all Che workspaces to volume mount their host Daemon when starting:
 CHE_WORKSPACE_VOLUME=/var/run/docker.sock:/var/run/docker.sock;
 ----
 
-1.  Configure Docker daemon to listen to listen to tcp socket and specify `DOCKER_HOST` environment variable in workspace machine. Each host environment will have different network topology/configuration so below is only to be used as general example. Configure your Docker daemon to listen on TCP. First, add the following to your Docker configuration file (on Ubuntu it’s `/etc/default/docker` - see the Docker docs for the location for your OS):
+* Configure the Docker daemon to listen to the TCP socket and specify the `DOCKER_HOST` environment variable in the workspace machine. Each host environment will have different network topology and configuration. The following is an example. 
+
+To configure your Docker daemon to listen on TCP:
 
-Second, export `DOCKER_HOST` variable in your workspace. You can do this in the terminal or make it permanent by adding `ENV DOCKER_HOST=tcp://$IP:2375` to a workspace recipe, where `$IP` is your docker daemon machine IP.
+. Add the following to your Docker configuration file (on Ubuntu: `/etc/default/docker`. See the Docker docs for the location for your OS):
```

Comment:
Add what to the docker config file? There seems to be nothing following after the : , missing information?

Reply:
Changed to:
To configure your Docker daemon to listen on TCP:

. Add the following to your Docker configuration file (on Ubuntu, the file is located at `/etc/default/docker`. For location of the Docker configuration file for your OS, see the https://docs.docker.com/engine/reference/commandline/dockerd/[Docker documentation]):
+
----
# Listen using the default unix socket, and on specific IP address on host.
# This will vary greatly depending on your host OS.
sudo dockerd -H unix:///var/run/docker.sock -H tcp://0.0.0.0:2375
# Verify that the Docker API is responding at: http://$IP:2375/containers/json
----
+
. Export the `DOCKER_HOST` variable in your workspace. You can do this in the terminal or make it permanent by adding `ENV DOCKER_HOST=tcp://$IP:2375` to a workspace recipe; here, `$IP` is your Docker daemon machine IP.
+
----
# In workspace machine
docker -H tcp://$IP:2375 ps

# Shorter form
export DOCKER_HOST="tcp://$IP:2375"
docker ps
----

Reply:
Edited.